tools/iea43_wra_data_model_postgresql.sql

```diff
@@ -543,8 +543,11 @@ CREATE TABLE IF NOT EXISTS calibration(
     sensor_uuid UUID NOT NULL,
     measurement_type_id text NOT NULL,
     slope decimal,
+	slope_unit decimal,
```

Comment:
@kersting  the columns for `slope_unit` or `offset_unit` should be varchar since they are not numbers

Reply:
I think we use 'text' mostly in the SQL statements.